### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by TrevorJoelHarris**
For more information about how to contribute to this repo, visit [this page](https://github.com/OfficeDev/microsoft-teams-library-js/blob/main/CONTRIBUTING.md).

## Description

Added information on what orgs internal employees need to join in order to contribute to teamsjs successfully

### Main changes in the PR:

1. Updated CONTRIBUTING file with instructions on what orgs internal Microsoft employees need to join

### Validation performed:

N/A

### Unit Tests added:

No

### End-to-end tests added:

No

### Change file added:

No

### CODE REVIEW

**Review by jadahiya-MSFT — DISMISSED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -17,7 +17,8 @@ Please review the [architectural guidelines](https://github.com/OfficeDev/micros
...
 3. Commit your changes.
 4. Run `pnpm changefile`, answer the following prompts, and push your changes to GitHub. **Important! Our pipeline checks will fail if you skip this step.** More on this below in our Change Log section.
-5. Create a [pull request against the `main` branch](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests). If you don't know what a pull request is read [this article](https://help.github.com/articles/using-pull-requests).
+5. If you are an internal Microsoft employee, you must join these organizations in order to have your changes built and tested in the pipeline: [Microsoft](https://repos.opensource.microsoft.com/) / [OfficeDev](https://repos.opensource.microsoft.com/orgs/OfficeDev/join) / [Teams Client SDK Contributors](https://github.com/orgs/OfficeDev/teams/teams-client-sdk-contributors)
```
Comment (erinha) [inline CONTRIBUTING.md:20]: We have a Q about Microsoft employees needing to request certain access. Should we move this info to that section and link to it here? Seems a bit harder to follow having access information in separate spots.

**Review by erinha — CHANGES_REQUESTED** [review]
Suggest organizing this info slightly differently

File: CONTRIBUTING.md (hunk 1 above)
Comment (TrevorJoelHarris) [reply CONTRIBUTING.md:20]: Ah, missed that, great, will update there

**Review by erinha — APPROVED** [review]
🚀

**Pull request merged**